components/apps_form/apps_form_component.scss

```diff
@@ -19,3 +19,44 @@
 .apps-form-modal-body-loading {
     z-index: 1000;
 }
+
+    .select_option_item {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        column-gap: 1.2rem;
+    }
+
+#apps_form_select_field{
+    .select_option__item {
```

Comment:
The rest of the css classes in this file have dashes instead of underscores. Can we make them consistent by making these use dashes?